### DISCUSSION TITLE
Several new Amazfit products with Zepp OS 2.1 released, simulator released simultaneously

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#202] [Announcements]
<img width="1468" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/73768aa7-6420-4318-95f9-57009b64cc0b">

<img width="1273" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/0840f543-5fb5-4003-a266-43c98ccb54ac">

![image](https://github.com/zepp-health/zeppos-forum/assets/107526244/9251a1a1-2ac2-44a9-8596-92d83b9eb60e)

At the same time, Amazfit GTS4 upgrades Zepp OS 2.1, OTA is being pushed one after another, and the simulator is ready.

**Comment by enxg** [comment]
Is GTR Mini going to receive 2.1?

**Reply by liqin-ZeppHealth** [reply]
we are trying to also upgrade 2.1 for GTR mini, but it needs sometime and we can't give a specific time currently, Please be patient and let's look forward it

**Comment by ThinkAndFly** [comment]
what about GTR2? should I get the GTR4 in order to be able to reply to WhatsApp?

**Reply by zdunex25** [reply]
GTR4 will not let you to respond by default anyways, moreover GTR2 does not support ZeppOS.
